server/routes/images.go

```diff
@@ -182,3 +196,44 @@ func UploadImage(ctx context.Context, user_id, image_type string, labels []model
 
 	return imageURL, nil
 }
+
+func GetUserImages(w http.ResponseWriter, r *http.Request) {
+	ctx := appengine.NewContext(r)
+	dsClient := services.Locator.DsClient()
+	vars := mux.Vars(r)
+	userID := vars["id"]
+
+	var imageURLs []string
+
+	if len(userID) == 0 {
+		w.WriteHeader(http.StatusBadRequest)
+		w.Write([]byte("no ID provided"))
+		return
+	}
+
+	query := datastore.NewQuery("Image").Order("-created").Filter("owner_id =", userID).Limit(100)
+	it := dsClient.Run(ctx, query)
+
+	for {
+		var image models.Image
+		_, err := it.Next(&image)
+		if err == iterator.Done {
+			break
+		}
+		if err != nil {
+			log.Errorf(ctx, "Error fetching next image: %v", err)
+			w.WriteHeader(http.StatusInternalServerError)
+			return
+		}
+		imageURLs = append(imageURLs, "https://storage.cloud.google.com/"+config.ImageBucket()+"/"+image.GCSObjectID())
```

Comment:
Consider using consts for magic strings (like URL in this case)

Reply:
Actually, I feel like making such strings is a responsibility of the StorageClient and you should add smth like `GetServingURL(bucket, object)` to the interface